docs/content/preview/yugabyte-voyager/reference/bulk-data-load/import-data-file.md

```diff
@@ -52,7 +52,7 @@ The valid *arguments* for import data file are described in the following table:
 | [--target-ssl-cert](../../yb-voyager-cli/#ssl-connectivity) <certificateName> | Path to a file containing the certificate which is part of the SSL `<cert,key>` pair. |
 | [--target-ssl-key](../../yb-voyager-cli/#ssl-connectivity) <keyName> | Path to a file containing the key which is part of the SSL `<cert,key>` pair. |
 | [--target-ssl-crl](../../yb-voyager-cli/#ssl-connectivity) <path> | Path to a file containing the SSL certificate revocation list (CRL).|
```

Comment:
This table has a lot of text inside angle brackets that does not get rendered on the page eg `<path>` presumably this is to indicate what the flag takes. angle brackets aren't what we use for that, assuming we want to see it on the page

Maybe use some hard-hyphens on one of the longer flag names for nicer line breaks